.gitignore

```diff
@@ -0,0 +1,5 @@
+__pycache__/
+cogs/__pycache__/
+config/
+data/raid_cache.json
+util/__pycache__/
```

Comment:
You can replace lines 1,2, & 5 with:
```
**/__pycache__
```

Reply:
Adjusted and will show on next push. Working on something with Andi before pushing again, explanation on push comment. Thanks!